pkg/assets/static/landingpage.js

```diff
@@ -213,15 +220,15 @@ function updateSignupState() {
 }
 
 function stopPolling() {
-  console.log('start polling..');
+  console.log('stop polling..');
   if (intervalRef) {
     clearInterval(intervalRef);
     intervalRef = undefined;
   }
 }
 
 function startPolling() {
-  console.log('stop polling..');
+  console.log('start polling..');
```

Comment:
rofl

Reply:
I was very confused. I had to look at the stopPolling and startPolling at least 3 times because I was sure I was seeing something wrong. Actually I'm still not sure 😂